src/algebra/module/basic.lean

```diff
@@ -325,6 +336,18 @@ begin
   { rw [int.neg_succ_of_nat_coe, neg_smul, neg_smul, nat_smul], }
 end
 
+local attribute [instance] int_module add_comm_monoid.nat_semimodule
+
+instance int_is_scalar_tower [ring S] [module S M] :
```

Comment:
Note that in the other file, the requirement was the stricter `[comm_ring S] [comm_ring M] [algebra S M]`